[2024-05-18T23:52:11.773+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-18T23:52:18.360+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Real_Estate_Webscraping_ETL.Process_and_Send_Data_to_DB scheduled__2024-05-18T12:05:00+00:00 [queued]>
[2024-05-18T23:52:18.808+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Real_Estate_Webscraping_ETL.Process_and_Send_Data_to_DB scheduled__2024-05-18T12:05:00+00:00 [queued]>
[2024-05-18T23:52:18.861+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 6
[2024-05-18T23:52:19.346+0000] {taskinstance.py:2330} INFO - Executing <Task(SparkSubmitOperator): Process_and_Send_Data_to_DB> on 2024-05-18 12:05:00+00:00
[2024-05-18T23:52:19.991+0000] {standard_task_runner.py:63} INFO - Started process 191 to run task
[2024-05-18T23:52:20.142+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Real_Estate_Webscraping_ETL', 'Process_and_Send_Data_to_DB', 'scheduled__2024-05-18T12:05:00+00:00', '--job-id', '373', '--raw', '--subdir', 'DAGS_FOLDER/webscrapping.py', '--cfg-path', '/tmp/tmpzoknm6m2']
[2024-05-18T23:52:20.289+0000] {standard_task_runner.py:91} INFO - Job 373: Subtask Process_and_Send_Data_to_DB
[2024-05-18T23:52:22.899+0000] {task_command.py:426} INFO - Running <TaskInstance: Real_Estate_Webscraping_ETL.Process_and_Send_Data_to_DB scheduled__2024-05-18T12:05:00+00:00 [running]> on host 4801ac514ca9
[2024-05-18T23:52:29.833+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='svc' AIRFLOW_CTX_DAG_ID='Real_Estate_Webscraping_ETL' AIRFLOW_CTX_TASK_ID='Process_and_Send_Data_to_DB' AIRFLOW_CTX_EXECUTION_DATE='2024-05-18T12:05:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-18T12:05:00+00:00'
[2024-05-18T23:52:30.119+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-18T23:52:33.859+0000] {base.py:84} INFO - Using connection ID 'spark_default' for task execution.
[2024-05-18T23:52:34.054+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --packages org.apache.spark:spark-sql-kafka-0-10_2.12:3.5.1,com.datastax.spark:spark-cassandra-connector_2.12:3.5.0,org.apache.kafka:kafka-clients:3.5.1,commons-logging:commons-logging:1.2 --name arrow-spark --deploy-mode client dags/includes/jobs/spark_consumer.py
[2024-05-18T23:52:38.919+0000] {spark_submit.py:571} INFO - /home/***/.local/lib/python3.11/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2024-05-18T23:55:34.471+0000] {spark_submit.py:571} INFO - :: loading settings :: url = jar:file:/home/***/.local/lib/python3.11/site-packages/pyspark/jars/ivy-2.5.1.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2024-05-18T23:55:56.621+0000] {local_task_job_runner.py:124} ERROR - Received SIGTERM. Terminating subprocesses
[2024-05-18T23:55:58.081+0000] {process_utils.py:132} INFO - Sending 15 to group 191. PIDs of all processes in the group: [202, 191]
[2024-05-18T23:55:58.175+0000] {process_utils.py:87} INFO - Sending the signal 15 to group 191
[2024-05-18T23:55:58.646+0000] {taskinstance.py:2611} ERROR - Received SIGTERM. Terminating subprocesses.
[2024-05-18T23:55:59.200+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=202, status='terminated', started='23:52:33') (202) terminated with exit code None
[2024-05-18T23:55:59.343+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-18T23:56:00.984+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 490, in submit
    self._process_spark_submit_log(iter(self._submit_sp.stdout))  # type: ignore
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 539, in _process_spark_submit_log
    for line in itr:
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 2613, in signal_handler
    raise AirflowTaskTerminated("Task received SIGTERM signal")
airflow.exceptions.AirflowTaskTerminated: Task received SIGTERM signal
[2024-05-18T23:56:02.049+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=Real_Estate_Webscraping_ETL, task_id=Process_and_Send_Data_to_DB, run_id=scheduled__2024-05-18T12:05:00+00:00, execution_date=20240518T120500, start_date=20240518T235218, end_date=20240518T235601
[2024-05-18T23:56:04.320+0000] {process_utils.py:80} INFO - Process psutil.Process(pid=191, status='terminated', exitcode=2, started='23:52:19') (191) terminated with exit code 2
[2024-05-18T23:56:04.349+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 143
[2024-05-18T23:56:04.898+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-18T23:56:04.923+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
